src/ansys/fluent/core/examples/downloads.py

```diff
@@ -131,3 +132,11 @@ def download_file(
 
     url = _get_file_url(filename, directory)
     return _retrieve_file(url, filename, save_path, return_only_filename)
+
+
+def path(filename: str):
+    file_path = str(Path(pyfluent.EXAMPLES_PATH) / filename)
+    if os.path.isfile(file_path):
+        return file_path
+    else:
+        raise FileNotFoundError(f"{filename} does not exist.")
```

Comment:
```suggestion
        raise FileNotFoundError(f"{file_path} does not exist.")
```
Small suggestion to make this exception statement more accurate (and informative)

Reply:
The message with `filename` is concise than `file_path`.